config/routes.rb

```diff
@@ -1,3 +1,19 @@
+# frozen_string_literal: true
+
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  # Zipcodes routes
```

Comment:
Please translate the routes to `resources` using the rails convention - https://guides.rubyonrails.org/routing.html#resources-on-the-web

Reply:
Done!